### PR TITLE
TSK-212: add notification on issue created

### PR DESCRIPTION
**Pull request by kristina-fefelova**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: packages/ui/src/components/notifications/NotificationPosition.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export enum NotificationPosition {
+  BottomLeft= 'BottomLeft',
```
Comment (haiodo) [inline packages/ui/src/components/notifications/NotificationPosition.ts]: I do not think we need string values for enums

Comment (kristina-fefelova) [reply packages/ui/src/components/notifications/NotificationPosition.ts]: Done

File: packages/ui/src/components/notifications/Notifications.svelte [hunk 2]
```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  import { setContext } from 'svelte'
+
+  import Notification from './Notification.svelte'
+  import store, { notificationsContextKey } from './store'
+  import { NotificationPosition } from './NotificationPosition'
+
+  const getClass = (position: NotificationPosition) => {
```
Comment (haiodo) [inline packages/ui/src/components/notifications/Notifications.svelte]: You could use record with positions.
```suggestion
const toPosition = {
  [NotificationPosition.BottomLeft]: 'bottom-left',
  [NotificationPosition.BottomRight]: 'bottom-right',
  [NotificationPosition.TopLeft: 'top-left',
  [NotificationPosition.TopRight]: 'top-right'
   }
```

File: packages/ui/src/components/notifications/Notifications.svelte [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        return 'top-right'
+    }
+  }
+
+  setContext(notificationsContextKey, store)
+</script>
+
+<slot />
+<div class="notifications">
+  {#each [NotificationPosition.TopRight, NotificationPosition.TopLeft, NotificationPosition.BottomRight, NotificationPosition.BottomLeft] as position}
```
Comment (haiodo) [inline packages/ui/src/components/notifications/Notifications.svelte]:
```suggestion
  {#each Object.enties(toPosition) as position}
```

File: packages/ui/src/components/notifications/store.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+const store = writable<Notification[]>([])
+
+interface NotificationsContext {
+  subscribe: typeof store.subscribe
+  addNotification: (notification: Notification) => void
+  removeNotification: (notificationId: string) => void
+}
+
+export const notificationsContextKey = 'notifications-context'
+export const getNotificationsContext = (): NotificationsContext => getContext(notificationsContextKey)
```
Comment (haiodo) [inline packages/ui/src/components/notifications/store.ts]: I do not think we need setContext/getContext here. You could just import store and use it as $store in Svelte.

File: packages/ui/src/components/notifications/Notifications.svelte (hunk 3 above)
Comment (kristina-fefelova) [reply packages/ui/src/components/notifications/Notifications.svelte]: Done.

File: packages/ui/src/components/notifications/store.ts (hunk 4 above)
Comment (kristina-fefelova) [reply packages/ui/src/components/notifications/store.ts]: Done.

File: packages/ui/src/components/notifications/Notifications.svelte (hunk 2 above)
Comment (kristina-fefelova) [reply packages/ui/src/components/notifications/Notifications.svelte]: Done.

**Review by haiodo — APPROVED** [review]

**Comment by haiodo** [comment]
@kristina-fefelova Please sign all commits. Thanks.

**Pull request merged**